### ISSUE TITLE
Add `%{app-exec-path}` and `%{app-bundle-path}` interpolation variable to `--format` flag in query commands

### ISSUE THREAD

**Issue by nikitabobko** [nikitabobko/AeroSpace#592]
Query commands to add it to:
- `list-apps`
- `list-windows`

```
$ aerospace list-apps --format '%{app-exec-path}'
/Applications/Google Chrome.app/Contents/MacOS/Google Chrome
/System/Library/CoreServices/Finder.app/Contents/MacOS/Finder

$ aerospace list-apps --format '%{app-bundle-path}'
/Applications/Google Chrome.app
/System/Library/CoreServices/Finder.app
```

Use case: Get app icon for spotlight-like applications https://github.com/raycast/extensions/pull/14908/files#diff-bb1e762a64d9a33559e2e0481c4980cea9f69d54fa3dcc1fd9e5fac95e0bb5e5R21

**Comment by nikitabobko** [comment]
Related: #359

**Comment by nikitabobko** [comment]
Released in 0.16.0-Beta
